htdocs/product/stock/movement_card.php

```diff
@@ -49,7 +49,7 @@
 // Security check
 $result = restrictedArea($user, 'stock');
 
-$id = GETPOST('id', 'int');
+$entrepot_id = GETPOST('id', 'int');
```

Comment:
It appears that the movement_card.php is a page view dedicated for 1 warehouse. So id is a werehouse and like for other pages the main object of the page is represented with $id for the id and $object for the main page. It allows to use shared common include files. What is the goal of renaming here into another var ?

Reply:
I changed the name of the variable because the central point of this page is the stock movement and not the warehouse itself. Also, several variables were previously used for different objects and renaming some variables makes it easier to understand the source code.